app/gen_installer_cmd.go

```diff
@@ -0,0 +1,85 @@
+package app
+
+import (
+	"bytes"
+	"crypto/sha256"
+
+	// Embed installer template
+	_ "embed"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/alecthomas/hcl"
+	"github.com/cashapp/hermit/errors"
+	"github.com/cashapp/hermit/state"
+)
+
+var (
+	//go:embed "install.sh.tmpl"
+	installerTemplateSource string
+	installerTemplate       = template.Must(template.New("install.sh").Funcs(template.FuncMap{
+		"string": func(b []byte) string { return string(b) },
+		"words":  func(s []string) string { return strings.Join(s, " ") },
+	}).Parse(installerTemplateSource))
+)
+
+type genInstallerCmd struct {
+	Schema bool   `help:"Display the global schema."`
```

Comment:
This flag is simply copied from `cmd/geninstaller`. I'm not sure if it is still useful or not. Should be easy to remove if it's no longer needed (the flag seems broken atm).